createdisk.sh

```diff
@@ -64,11 +54,10 @@ function resize_and_sparsify {
     fi
 
     guestfish --remote <<EOF
-add-drive $tmpFile
+add-drive $baseDir/$srcFile
 run
 luks-open $partition coreos-root
 mount /dev/mapper/coreos-root /
-xfs-growfs /
 zero-free-space /boot/
 EOF
     if [ $? -ne 0 ]; then
```

Comment:
Just below that line, the error message is "Failed to resize and sparsify $baseDir/$srcFile, aborting", you can remove the "resize" mention  from it.